### ISSUE TITLE
[Request] Select and Move bookmarks into folders

### ISSUE THREAD

**Issue by Unknow0059** [go-shiori/shiori#248]
This is a basic feature all bookmark managers must have - being able to select and to move bookmarks into different folders.

shiori is one of the few promising offline bookmark managers - out of three, including itself - and it's the only one to currently have a usable non-list view, so I care about what this project could evolve into.

**Comment by deanishe** [comment]
> This is a basic feature all bookmark managers must have - being able to select and to move bookmarks into different folders.

Why? I mean, what do folders offer that tags don't? You can assign a bookmark to as many tags as you like. With folders, you're stuck with just one.

Could you describe what your actual end goal is rather than how you want to achieve that?

**Comment by Unknow0059** [comment]
I like Shiori's interface because it's grid-based. I can't organize my bookmarks if I can't manage them in folders because that's how browsers organize them and it's also how I interact with them.

Even if I started opening bookmarks through Shiori and ditched the folder-based browser interface - the tagging isn't robust either, seeing as I can't select multiple bookmarks and have to edit them money by one, though this part I already described.

**Comment by deanishe** [comment]
> because that's how browsers organize them and it's also how I interact with them

You're asking for a very big change here. I'm afraid I'll need a better argument that "app X does it that way". I still don't see how interacting with folders is meaningfully different to interacting with tags.

> seeing as I can't select multiple bookmarks

Yes, you can. Top right, pencil icon. "Batch edit".

**Comment by Unknow0059** [comment]
Indeed. The folder feature doesn't make any sense. So I'll drop that.

I didn't know the pencil icon was for Batch Edit. I thought it was for Single Edit (like in many other interfaces) so I never bothered to hover on it. Thanks, I can now consider using Shiori properly.

How about adding the ability to hold the mouse and select multiple entries at once? That would make the experience better.

**Comment by deanishe** [comment]
> How about adding the ability to hold the mouse and select multiple entries at once? That would make the experience better.

If you submit a PR, and it doesn't negatively affect usability on mobile, sure.

I can't really see myself implementing it because it would be a lot of effort for a very small improvement in usability.
